clang/include/clang/Format/Format.h

```diff
@@ -1208,6 +1208,21 @@ struct FormatStyle {
   /// \version 3.7
   bool BinPackArguments;
 
+  /// If ``BinPackArguments`` is ``false`` this option can override it if
+  /// ``true`` when 20 or more items are in a braced initializer list.
```

Comment:
```suggestion
  /// If ``BinPackLongBracedLists`` is set it overrides ``BinPackArguments``, if there are 20 or more items are in a braced initializer list.
```

How about that? The current sentence is hard to read.

Reply:
I rewrote it similar to your suggestion.